mlrun/model_monitoring/db/v3io_tsdb_reader.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Move this module into the TSDB abstraction once it is in.
+
+from datetime import datetime
+from io import StringIO
+
+import pandas as pd
+
+import mlrun
+import mlrun.common.schemas.model_monitoring.constants as mm_constants
+import mlrun.model_monitoring.writer as mm_writer
+import mlrun.utils.v3io_clients
+from mlrun.common.schemas.model_monitoring.model_endpoints import (
+    ModelEndpointMonitoringMetric,
+    ModelEndpointMonitoringMetricType,
+    ModelEndpointMonitoringResultNoData,
+    ModelEndpointMonitoringResultValues,
+    _compose_full_name,
+    _ModelEndpointMonitoringResultValuesBase,
+)
+from mlrun.model_monitoring.db.stores.v3io_kv.kv_store import KVStoreBase
+from mlrun.model_monitoring.db.tsdb.v3io.v3io_connector import _TSDB_BE
```

Comment:
I guess here too, if we need to import it here, maybe it shouldn't be private.

Reply:
You're right, but this module should move entirely to `mlrun.model_monitoring.db.tsdb.v3io.v3io_connector`, so this import won't happen :)